src/index.js

```diff
@@ -70,17 +70,16 @@ export default async function loader(content, sourceMap, meta) {
   );
 
   if (useSourceMap) {
-    processOptions.map = { inline: false, annotation: false };
+    if (
+      typeof processOptions.map === 'undefined' ||
+      typeof processOptions.map === 'boolean'
+    ) {
+      processOptions.map = { inline: false, annotation: false };
+    }
 
     if (sourceMap) {
       processOptions.map.prev = normalizeSourceMap(sourceMap, this.context);
     }
-  } else if (sourceMap && typeof processOptions.map !== 'undefined') {
-    if (typeof processOptions.map === 'boolean') {
-      processOptions.map = { inline: true };
-    }
-
-    processOptions.map.prev = sourceMap;
```

Comment:
Also it is breaking change, now you need always to set `sourceMap: true` if you need source maps, but it is invalid in some cases I don't need webpack source maps, I need only postcss source maps (for example generate library with css and relative urls in sources)

Reply:
Maybe I can amend the logic to check for either `useSourceMaps` or `postcssOptions.map`?

Reply:
What do you want to achieve? I don't understand what you are trying to do, if you want to generate inline source map just set `postcssOptions.map: true`

Reply:
At the moment if devtool is set you cannot “inline” a sourcemap.